### PR TITLE
[query] Table sample data preview incorrectly rounds bigint value

### PR DESCRIPTION
**Pull request by Akhilsnaik**
**What changes were proposed in this pull request?**
https://github.com/cloudera/hue/issues/1679 - Table sample data preview incorrectly rounds bigint value

**How was this patch tested?**
-- Tested Manually whether the sample is showing correct datta or not after the fix.
Screenshot 2021-01-23 at 1 25 31 AM
Screenshot 2021-01-23 at 1 25 08 AM

### CODE REVIEW

**Review by romainr — COMMENTED** [review]
:+1: 

Nice change and not sure for the proper way to import in order to avoid the warnings

File: desktop/core/src/desktop/js/catalog/api.ts [hunk 1]
```diff
@@ -432,6 +432,11 @@ export const fetchSample = ({
         }
 
         snippetJson = JSON.stringify(queryResult);
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```
Comment (JohanAhlen) [inline desktop/core/src/desktop/js/catalog/api.ts:436]: I'll take care of the ts-ignore and fix so we can do import bigdataParse.

**Review by JohanAhlen — APPROVED** [review]
Nice catch!

File: desktop/core/src/desktop/js/catalog/api.ts (hunk 1 above)
Comment (Akhilsnaik) [reply desktop/core/src/desktop/js/catalog/api.ts:436]: Merged to Master

**Pull request merged**